diff --git a/go.sum b/go.sum
index 0095dec..79ca4ae 100644
--- a/go.sum
+++ b/go.sum
@@ -1,4 +1,5 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0 h1:eOI3/cP2VTU6uZLDYAoic+eyzzB9YyGmJ7eIjl8rOPg=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da/go.mod h1:Q73ZrmVTwzkszR9V5SSuryQ31EELlFMUz1kKyl939pY=
diff --git a/pkg/provider/basic.go b/pkg/provider/basic.go
index 592ef24..606f1ab 100644
--- a/pkg/provider/basic.go
+++ b/pkg/provider/basic.go
@@ -8,6 +8,7 @@ import (
 	"golang.org/x/oauth2/bitbucket"
 	"golang.org/x/oauth2/github"
 	"golang.org/x/oauth2/gitlab"
+	"golang.org/x/oauth2/google"
 	"golang.org/x/oauth2/microsoft"
 	"golang.org/x/oauth2/slack"
 )
@@ -16,6 +17,7 @@ func init() {
 	GlobalRegistry.MustRegister("bitbucket", basicFactory(bitbucket.Endpoint))
 	GlobalRegistry.MustRegister("github", basicFactory(github.Endpoint))
 	GlobalRegistry.MustRegister("gitlab", basicFactory(gitlab.Endpoint))
+	GlobalRegistry.MustRegister("google", basicFactory(google.Endpoint))
 	GlobalRegistry.MustRegister("microsoft_azure_ad", azureADFactory)
 	GlobalRegistry.MustRegister("slack", basicFactory(slack.Endpoint))
 
